### PR TITLE
Nash_Searchbar finished

### PR DESCRIPTION
**Pull request by nash4comp**
- Created searchbar.js and put the related methods in there
- Search the keyword from the database in email, firstname, and lastname

### CODE REVIEW

File: public/js/searchbar.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  }
+
+  document.getElementById("searchButton").addEventListener("click", async function (e) {
+    var keyword = document.getElementById("searchKeyword").value;
+    let response = await postData("/getUsersKeyword", {
+      keyword: keyword
+    });
+
+    if (response.users.length == 0) {
+      alert("No search results found.");
```
Comment (mksample) [inline public/js/searchbar.js:37]: Aaron says we shouldn't use alerts. "Do not use the alert in JS ... for anything. Ever. Period."

File: public/css/admin_profile.css [hunk 2]
```diff
@@ -116,19 +116,10 @@ aside h1 {
     padding: 20px;
 }
 
-/* The Close Button */
```
Comment (mksample) [inline public/css/admin_profile.css:119]: Is there any reason you removed all the styling from the admin page? The create user and edit user modals are now blank. Also, the delete user button has disappeared and is unclickable now. This comment applies to the whole file.

**Review by mksample — CHANGES_REQUESTED** [review]

File: public/js/searchbar.js (hunk 1 above)
Comment (nash4comp) [reply public/js/searchbar.js:37]: Okay, I'll delete it.

File: public/css/admin_profile.css (hunk 2 above)
Comment (nash4comp) [reply public/css/admin_profile.css]: Recovered and adopted the similar style to searchbar.

File: public/js/searchbar.js (hunk 1 above)
Comment (nash4comp) [reply public/js/searchbar.js:37]: Replaced the alert to message warning below the searchbar.

**Review by mksample — APPROVED** [review]
LGTM!

**Pull request merged**